### ISSUE TITLE
Add in more precise audio settings

### ISSUE THREAD

**Issue by GameguySD** [Addikins/RPG-Tech-Demo#15]

**Comment by Addikins** [comment]
Can you explain this one more in-depth? Do you mean things such as volume control?

**Comment by GameguySD** [comment]
Yeah, volume control. Stuff like volume bars / meters, and maybe some sfx-specific audio toggles like "taking damage" or "attack sfx".
